docs/reference/configuration/Storage.md

```diff
@@ -1,21 +1,47 @@
 <h2 id="configuration-storage">Storage</h2>
 
+This page describes the build-time configurable parameters for storage in Mbed OS. There are no standard shared configuration options for different storage module. Instead, each module has its own implementation-specific set of configuration options.
+
+- [LittleFS configuration](#littlefs-configuration).
+- [NVStore configuration](#nvstore-configuration).
+
 The following is the complete list of storage configuration parameters, as generated by `mbed compile --config -v`. Please see [the configuration system documentation](/docs/development/reference/configuration.html) for details on how you may use or override these settings.
 
+### LittleFS configuration
+
+LittleFS provides several configuration options that you can use to tweak the performance of the file system on different hardware. By default, this file system finds the optimal configuration from the underlying block device's geometry, but you can override this to optimize special situations. For example, if your device has a large amount of RAM, you can increase the `read_size` and `prog_size` configuration options for a minor speed improvement.
+
+Note that LittleFS has 4 levels of debug logging. By default, all logging is enabled except for `enable_debug`. Setting `enable_debug` to `true` makes the log output very verbose, and the output is useful for bug reports.
+
 ```
 Configuration parameters
 ------------------------
 
-Name: filesystem.present
```

Comment:
Query: Does this config parameter still exist? If so, we should document it, and I can create an additional section for it.

Reply:
It's not a parameter that can be configured. It's really a workaround to get the build tools to indicate which libraries are available.

I noticed this nsapi.present config option was omitted from the network config, so I did the same here.